api/lib/application/users/index.js

```diff
@@ -135,13 +135,15 @@ exports.register = async function(server) {
           payload: {
             data: {
               attributes: {
-                password: Joi.string().regex(XRegExp(passwordValidationPattern)),
+                password: Joi.string().regex(XRegExp(passwordValidationPattern)).allow(null),
                 'pix-orga-terms-of-service-accepted': Joi.boolean(),
-                'pix-certif-terms-of-service-accepted': Joi.boolean()
-              }
+                'pix-certif-terms-of-service-accepted': Joi.boolean(),
+                'has-seen-migration': Joi.boolean(),
```

Comment:
Si `has-seen-migration`désigne bien le fait que l'utilisateur a vu la modale, je mettrais plutôt `is-aware-of-migration` ou `has-seen-migrartion-modal`

Reply:
`has-seen-migration-modal` me parait plus parlant; has-seen-migration pourrait vouloir dire plusieurs choses